UnityProject/Assets/Scripts/Core/Camera/CameraEffectControlScript.cs

```diff
@@ -27,9 +28,16 @@ public class CameraEffectControlScript : MonoBehaviour
 		private float drunkCameraTime = 0;
 
 		public LightingSystem LightingSystem;
+		
 		public void Awake()
 		{
 			LightingSystem = this.GetComponent<LightingSystem>();
+			if (minimalVisibilitySprite != null)
+			{
+				MinimalVisibilityScale = minimalVisibilitySprite.transform.localScale;
+				return;
+			}
+			Logger.LogWarning("[CameraEffectControlScript] - visibilitySprite is null! please set it from the inspector.");
```

Comment:
I think some things like xenos don't actually require it because it messes up some of their effects, so I don't think it should be a requirement for now.